docs/source/en/model_doc/vitpose.md

```diff
@@ -1,286 +1,151 @@
-                and score1 > keypoint_score_threshold
-                and score2 > keypoint_score_threshold
-            ):
-                color = tuple(int(c) for c in link_colors[sk_id])
-                if show_keypoint_weight:
-                    X = (x1, x2)
-                    Y = (y1, y2)
-                    mean_x = np.mean(X)
-                    mean_y = np.mean(Y)
-                    length = ((Y[0] - Y[1]) ** 2 + (X[0] - X[1]) ** 2) ** 0.5
-                    angle = math.degrees(math.atan2(Y[0] - Y[1], X[0] - X[1]))
-                    polygon = cv2.ellipse2Poly(
-                        (int(mean_x), int(mean_y)), (int(length / 2), int(stick_width)), int(angle), 0, 360, 1
-                    )
-                    cv2.fillConvexPoly(image, polygon, color)
-                    transparency = max(0, min(1, 0.5 * (keypoints[sk[0], 2] + keypoints[sk[1], 2])))
-                    cv2.addWeighted(image, transparency, image, 1 - transparency, 0, dst=image)
-                else:
-                    cv2.line(image, (x1, y1), (x2, y2), color, thickness=thickness)
-
-
-# Note: keypoint_edges and color palette are dataset-specific
-keypoint_edges = model.config.edges
-
-palette = np.array(
-    [
-        [255, 128, 0],
-        [255, 153, 51],
-        [255, 178, 102],
-        [230, 230, 0],
-        [255, 153, 255],
-        [153, 204, 255],
-        [255, 102, 255],
-        [255, 51, 255],
-        [102, 178, 255],
-        [51, 153, 255],
-        [255, 153, 153],
-        [255, 102, 102],
-        [255, 51, 51],
-        [153, 255, 153],
-        [102, 255, 102],
-        [51, 255, 51],
-        [0, 255, 0],
-        [0, 0, 255],
-        [255, 0, 0],
-        [255, 255, 255],
-    ]
-)
+np_img = np.array(image).copy()
+kpts = pose_results[0]["keypoints"]
+for x, y, score in kpts:
+    if score > 0.3:
+        cv2.circle(np_img, (int(x), int(y)), 3, (0, 255, 0), -1)
 
-link_colors = palette[[0, 0, 0, 0, 7, 7, 7, 9, 9, 9, 9, 9, 16, 16, 16, 16, 16, 16, 16]]
-keypoint_colors = palette[[16, 16, 16, 16, 16, 9, 9, 9, 9, 9, 9, 0, 0, 0, 0, 0, 0]]
+cv2.imshow("Pose", np_img)
+cv2.waitKey(0)
+```
 
-numpy_image = np.array(image)
+</hfoption>
+</hfoptions>
 
-for pose_result in image_pose_result:
-    scores = np.array(pose_result["scores"])
-    keypoints = np.array(pose_result["keypoints"])
+---
 
-    # draw each point on image
-    draw_points(numpy_image, keypoints, scores, keypoint_colors, keypoint_score_threshold=0.3, radius=4, show_keypoint_weight=False)
+## Notes
 
-    # draw links
-    draw_links(numpy_image, keypoints, scores, keypoint_edges, link_colors, keypoint_score_threshold=0.3, thickness=1, show_keypoint_weight=False)
+- ViTPose is a top-down pose estimator, meaning it requires a separate object detector to first locate individuals in the image.
+- Use `AutoProcessor` to automatically prepare bounding box and image inputs.
+- ViTPose++ supports passing a `dataset_index` parameter to specify which expert head to use.
```

Comment:
Add a code snippet here demonstrating using `dataset_index` with ViTPose++ models to fix #36773!